internal/testrunner/runners/pipeline/test_result.go

```diff
@@ -54,9 +61,10 @@ func compareResults(testCasePath string, result *testResult) error {
 
 	report := diff.Diff(string(expected), string(current))
```

Comment:
nit: I think the `current` word used to be known as `actual` (expected and actual).

Reply:
I didn't change it to `current` in this PR. It was like that before this PR. So I will fix it to `actual` in a separate PR as that change is unrelated to the changes in this PR.

Reply:
https://github.com/elastic/elastic-package/pull/142